test/util.jl

```diff
@@ -0,0 +1,12 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
```

Comment:
hmm, I thought each xxx.jl in `base` has a corresponding xxx.jl in test. Which file should it belong to then?